extras/kms/repository_data_key_version.go

```diff
@@ -165,7 +166,7 @@ func (r *repository) LatestDataKeyVersion(ctx context.Context, rkvWrapper wrappi
 }
 
 // ListDataKeyVersions will lists versions of a key. Supported options:
-// WithLimit, WithOrderByVersion
+// WithLimit, WithOrderByVersion, WithReader
 func (r *repository) ListDataKeyVersions(ctx context.Context, rkvWrapper wrapping.Wrapper, databaseKeyId string, opt ...Option) ([]*dataKeyVersion, error) {
 	const op = "kms.(repository).ListDataVersions"
```

Comment:
ListDataVersions vs ListDataKeyVersions

Reply:
🤔 not much better... might just keep it the same for now.

Reply:
I think @jefferai is calling out the mismatch between the function name and the `op` string.

Reply:
ty!

Reply:
@tmessi ding ding :-D